geomstats/hypersphere.py

```diff
@@ -236,14 +236,6 @@ def log(self, point, base_point):
         mask_0_float = gs.cast(mask_0, gs.float32)
         mask_else_float = gs.cast(mask_else, gs.float32)
 
-        angle_0 = gs.boolean_mask(angle, mask_0)
```

Comment:
why remove this?

Reply:
it was not used. left over from a previous version probably